homeassistant/helpers/config_validation.py

```diff
@@ -739,15 +739,7 @@ def template(value: Any | None) -> template_helper.Template:
     if isinstance(value, (list, dict, template_helper.Template)):
         raise vol.Invalid("template value should be a string")
     if not (hass := _async_get_hass_or_none()):
-        from .frame import ReportBehavior, report_usage  # noqa: PLC0415
-
-        report_usage(
-            (
-                "validates schema outside the event loop, "
-                "which will stop working in HA Core 2025.10"
-            ),
-            core_behavior=ReportBehavior.LOG,
-        )
+        raise vol.Invalid("Validate schema outside the event loop")
```

Comment:
This sounds like an encouragement to validate outside the event loop, but the correct way is the opposite.

Reply:
```suggestion
        raise vol.Invalid("Validates schema outside the event loop")
```